### PR TITLE
fix: add missing behavior when domain list is empty

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi team, 

I fixed this scenario:

image

I am creating a new task to improve it in the future.

### CODE REVIEW

File: src/components/Reports/Reports.js [hunk 1]
```diff
@@ -32,20 +33,18 @@ class Reports extends React.Component {
...
-      this.setState({
-        domains: domains,
-        domainSelected: domainSelected,
-        pages: pages,
-        pageSelected: pageSelected,
-        dateFrom: dateFrom,
-      });
-    }
+    const domainSelected = domains.length ? domains[0] : null;
+    const pages = [];
```
Comment (fcoronelmakingsense) [inline src/components/Reports/Reports.js]: no need and if here? if doesn't has domains not pages too..

Comment (andresmoschini) [reply src/components/Reports/Reports.js]: I am thinking about removing all pages related code by the moment...

Comment (andresmoschini) [reply src/components/Reports/Reports.js]: I think that this solution is ugly, but safe regarding to that, so, by the moment I will keep it as it is.

Thanks @fcoronelmakingsense !

**Pull request merged**